app/Http/Controllers/Remote/RemoteController.php

```diff
@@ -107,4 +108,30 @@ public function event(Request $request)
 
         return response('', 201);
     }
+
+    public function getConfiguration(Request $request, $tokenString)
+    {
+        // Try to query the token and the node from the database
+        try {
+            $token = Models\NodeConfigurationToken::where('token', $tokenString)->firstOrFail();
```

Comment:
I think we could include the date/time checking in this query as well to avoid doing it later.